### PR TITLE
Added ng-file-upload

### PR DESCRIPTION
**Pull request by whitef0x0**

### CODE REVIEW

**Comment by whitef0x0** [comment]
@jsdelivrbot

File: files/ng-file-upload/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Danial Farid"
+github = "https://github.com/danialfarid/ng-file-upload"
+homepage = "https://github.com/danialfarid/ng-file-upload"
+description = "An AngularJS directive for file upload using HTML5 with FileAPI polyfill for unsupported browsers"
+mainfile = "index.js"
```
Comment (jimaek) [inline files/ng-file-upload/info.ini]: `ng-file-upload-all.min.js`

File: files/ng-file-upload/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "ng-file-upload",
+  "repo": "danialfarid/ng-file-upload",
+  "files": {
+    "include": ["ng-file-upload-shim.min.js", "ng-file-upload-all.min.js", "ng-file-upload.min.js", "FileAPI.min.js"]
```
Comment (jimaek) [inline files/ng-file-upload/update.json]: These files are located inside the `dist/` folder. Please fix the paths

Comment (jimaek) [inline files/ng-file-upload/update.json]: Also I believe `FileAPI.flash.swf` is required as well in production. is it not?

**Comment by jimaek** [comment]
Actually sorry, we already host this project http://www.jsdelivr.com/projects/angular.file-upload

**Comment by DiegoRBaquero** [comment]
@jimaek It's outdated (the version you host). The docs on project's github don't apply for that version. So it's pretty much useless.

**Pull request closed without merging**